roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -35,7 +35,11 @@ oom_score = {{ containerd_oom_score }}
 
           [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.{{ runtime.name }}.options]
 {% for key, value in runtime.options.items() %}
+{% if value|bool %}
```

Comment:
Here you wanted to check if it's a boolean to not add quote right?

Reply:
The systemdCgroup is boolean, it cannot be quoted. And the binaryName var is string type. 
They need to be treated separatorly. Otherwise it will raise error

Reply:
I am pretty sure this doesn't work then, what if the value is supposed to be false for instance? It will be converted to a boolean and will go into the else...

Reply:
OK. I'll check the var is boolean type or not. Thank you~